modules/swagger-codegen/src/main/resources/JavaJaxRS/apiServiceImpl.mustache

```diff
@@ -21,13 +21,14 @@ import javax.ws.rs.core.Response;
 {{>generatedAnnotation}}
 {{#operations}}
 public class {{classname}}ServiceImpl extends {{classname}}Service {
-  {{#operation}}
-      @Override
-      public Response {{nickname}}({{#allParams}}{{>serviceQueryParams}}{{>servicePathParams}}{{>serviceHeaderParams}}{{>serviceBodyParams}}{{>serviceFormParams}}{{#hasMore}},{{/hasMore}}{{/allParams}})
-      throws NotFoundException {
-      // do some magic!
-      return Response.ok().entity(new ApiResponseMessage(ApiResponseMessage.OK, "magic!")).build();
-  }
-  {{/operation}}
+{{#operation}}
+
+    @Override
+    public Response {{nickname}}({{#allParams}}{{>serviceQueryParams}}{{>servicePathParams}}{{>serviceHeaderParams}}{{>serviceBodyParams}}{{>serviceFormParams}}{{#hasMore}}, {{/hasMore}}{{/allParams}})
+    throws NotFoundException {
+        // do some magic!
```

Comment:
I'd suggest using `// TODO: Implement this` or `// TODO: Do some magic`, so findbugs or IntelliJ would warn you if you committing code with new TODOs.